commit 5c29266959a764983b50a1901926c87c9d2ce7a6 (refs/changes/88/2388/1)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2017-08-14T18:58:24+00:00 (2 years, 2 months ago)
    
    MANTA-2500 muskie default content length is not one size fits all

diff --git a/config/services/webapi/service.json.coal b/config/services/webapi/service.json.coal
index 5677ff1..fd56d86 100644
--- a/config/services/webapi/service.json.coal
+++ b/config/services/webapi/service.json.coal
@@ -1,5 +1,5 @@
 {
 	"metadata": {
-		"MUSKIE_IGNORE_SIZE": true
+		"MUSKIE_DEFAULT_MAX_STREAMING_SIZE_MB": 5120
 	}
 }
diff --git a/config/services/webapi/service.json.lab b/config/services/webapi/service.json.lab
index 9154a7f..84eceff 100644
--- a/config/services/webapi/service.json.lab
+++ b/config/services/webapi/service.json.lab
@@ -1,5 +1,8 @@
 {
 	"params": {
 		"ram": 1536
+	},
+	"metadata": {
+		"MUSKIE_DEFAULT_MAX_STREAMING_SIZE_MB": 5120
 	}
 }
